client.go

```diff
@@ -121,10 +128,31 @@ func (c *Client) NewRequest(method, path string, body interface{}) (*http.Reques
 	req.SetBasicAuth("token", c.token)
 	req.Header.Set("Accept", defaultMediaType)
 	req.Header.Set("Content-Type", defaultMediaType)
+	req.Header.Set("User-Agent", c.userAgentString)
 
 	return req, nil
 }
 
+// setUserAgent is a config function allowing setting of the User-Agent header in requests
+func setUserAgent(userAgentString string) func(*Client) error {
```

Comment:
I might be missing it, but how do I set this option from outside the package?

Reply:
(comment applies to both options obviously)

Reply:
@snorecone you're not missing it. I was smoking it. Fixed. 

See? See? **This** is why we have code review. Lmfao. 🤦‍♂️

Reply:
Haha, I feel you!